cassiopeia/spring/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM frolvlad/alpine-oraclejdk8
```

Comment:
I don't feel fully qualified to review this Dockerfile for completeness, but at a glance, it appears syntactically correct and easy to follow.

Reply:
I took this directly from orion, so I know it works.

We could probably pull the spring container out and create an image that we use rather than duplicating.

Reply:
That's what I gathered.  I have no issues then.  Thank you!